sys/include/net/gnrc/nettype.h

```diff
@@ -8,7 +8,7 @@
 
 /**
  * @defgroup    net_gnrc_nettype  Protocol type
- * @ingroup     net
+ * @ingroup     net_gnrc
  * @brief       Protocol type definitions to be used with the @ref net_gnrc_netapi,
  *              the @ref net_gnrc_netdev, the @ref net_gnrc_netreg, and
  *              the @ref net_gnrc_pkt
```

Comment:
Line 12-14: I love those references (surprise!) but still it might be enough to have them in the *Detailed description*. And there are helper functions. Can you agree to this change?

     * @brief       Protocol type definitions and helper functions.
     *              To be used with the @ref net_gnrc_netapi,
     *              the @ref net_gnrc_netdev, the @ref net_gnrc_netreg, and
     *              the @ref net_gnrc_pkt.